src/js/libgeo/GeoOps.js

```diff
@@ -625,6 +625,144 @@ geoOps.PointOnSegment.updatePosition = function(el) {
 };
 geoOps.PointOnSegment.stateSize = 2;
 
+geoOps._helper.PointOnArcCr = function(arc,P){
+    var A = arc.startPoint, B = arc.viaPoint, C = arc.endPoint;
+    //var cr = List.crossratio3harm(P,A,B,C,List.ii);
+    var cr = List.crossratio3harm(A,C,B,P,List.ii);
+    var m = cr.value[0];
+    var n = cr.value[1];
+    if (!CSNumber._helper.isAlmostZero(m)) {
+        n = CSNumber.div(n, m);
+        m = CSNumber.real(1);
+    } else {
+        m = CSNumber.div(m, n);
+        n = CSNumber.real(1);
+    }
```

Comment:
Yes that's true and i'm not quite sure why this happens. I'm using m and n later on separately (to avoid the infinity case). I'll look into that.